pages/calendar/index.js

```diff
@@ -0,0 +1,19 @@
+import Calendar from "@/components/Calendar";
+import Heading from "@/components/Heading";
+import styled from "styled-components";
+
+export default function CalendarPage({ emotions }) {
+  const createdEmotions = emotions.filter((emotion) => emotion.id);
```

Comment:
I refactored it without a filter method. I checked and since we are not using a database yet it is not necessary to have an id because they are used only locally.

Great hint!